2529_cpp_wodnjs.cpp

```diff
@@ -0,0 +1,83 @@
+#include<iostream>
+
+using namespace std;
+int arrange[10];
+char inequ[9];
+bool used[10];
+long long minr = 9876543211;
+int min_arr[10];
+long long maxr = 0;
+int max_arr[10];
+void make_arr(int idx,int total);
+int main() {
+	ios::sync_with_stdio(false);
+	cin.tie(nullptr);
+	cout.tie(nullptr);
+	int _num;
+	cin >> _num;
+	for (int n = 0; n < _num; n++)
+	{
+		cin >> inequ[n];
+	}
+	make_arr(0,_num);
+	for (int i = 0; i < _num + 1; i++)
+	{
+		cout << max_arr[i];
+	}
+	cout << '\n';
+	for (int i = 0; i < _num + 1; i++)
+	{
+		cout << min_arr[i];
+	}
+	cout << '\n';
+	return 0;
+}
+void make_arr(int idx,int _num) {
+	if (idx == _num+1)
+	{
+		for (int i = 0; i < _num; i++)
+		{
+			if (inequ[i] == '>')
+			{
+				if (arrange[i] < arrange[i + 1])
+					return;
+			}
+			if (inequ[i] == '<')
+			{
+				if (arrange[i] > arrange[i + 1])
+					return;
+			}
```

Comment:
그런데 사실 ||까지 이용하면 더 깔끔하게 쓸 수 있습니다.
```suggestion
			if ((inequ[i] == '>' && arrange[i] < arrange[i + 1]) || (inequ[i] == '<' && arrange[i] > arrange[i + 1]))
					return;
```